src/app/components/InRadius/InRadius.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+
+type InRadiusProps = {
+  getRadius: (radius: number) => void
+}
+
+export default function InRadius({ getRadius }: InRadiusProps): JSX.Element {
+  const [radius, setRadius] = useState(0)
+
+  getRadius(radius)
```

Comment:
In the future, it's probably best to wrap this in a condition and/or maybe use `useEddect`?